docs/hosting-a-server-with-northstar/guides/pg9182's-Docker-Installation-Guide.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
Because gitbook has some weird requirements, basically the first line should be the title of the page as a top header (i.e. single `#`) which should also be the same as the filename to some degree.

All headers, should be `##` or smaller cause GitBook™ ¯\\\_(ツ)\_/¯